src/style/blocks/card.css

```
@@ -0,0 +1,94 @@
+:root {
+  --main-color: #060b35;
+  --secondary-color: #616070;
+  --accent-color: #00acdc;
+}
+
+body {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-color);
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  margin: 0 auto;
```

So in this way your paddings would be as a waterfall (all go down)
```suggestion
  margin: 0 auto 40px;
```